src/components/TestnetNotification.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { Alert } from './Alert';
+
+const StyledAlert = styled(Alert)`
+  text-align: center;
+`;
+
```

Comment:
(If going with the suggestion below)
```suggestion

const AllCaps = styled.span`
  text-transform: uppercase;
`;

```